src/js/comments.js

```diff
@@ -24,26 +24,37 @@ class Comments {
 	 */
 	_renderComments () {
 		/*global Coral*/
-		const scriptElement = document.createElement('script');
-		scriptElement.src = 'https://ft-next-talk-spike.herokuapp.com/static/embed.js';
-		scriptElement.onload = () => Coral.Talk.render(document.querySelector('[data-o-component="o-comments"]'),
-			{
-				talk: 'https://ft-next-talk-spike.herokuapp.com',
-				events: (events) => {
-					events.onAny((name, data) => {
-						const message = new CustomEvent('talkEvent', {
-							detail: {
-								name,
-								data
-							}
-						});
-						parent.dispatchEvent(message);
-					});
-				}
-			}
-		);
-
-		this.oCommentsEl.appendChild(scriptElement);
+		fetch(`https://comments-auth.ft.com/v1/jwt/`, {
```

Comment:
I think we might need to include `fetch` in the origami.json file so it gets pulled in by the polyfill. @Financial-Times/origami-core would one of you mind confirming if I am right about this?

This is where it would go https://github.com/Financial-Times/o-comments/blob/v6/origami.json#L15

Reply:
You are correct

Reply:
Done 😄